lib/concentrate/encoder/gtfs_realtime_helpers.ex

```diff
@@ -140,6 +140,16 @@ defmodule Concentrate.Encoder.GTFSRealtimeHelpers do
   def schedule_relationship(:SCHEDULED), do: nil
   def schedule_relationship(relationship), do: relationship
 
+  @doc """
+  Returns true if the group is non-revenue
+  """
+  def is_non_revenue?({td, _, stus} = _group) do
```

Comment:
suggestion(nitpick):
```suggestion
  def non_revenue?({td, _, stus} = _group) do
```

question: I thought the plan was to provide a flag in the output to indicate that a trip was non-revenue, rather than parsing the trip IDs?

Reply:
There's discussion around the flag happening in Slack currently, but given that this fixes us sharing information we currently say we aren't, I don't think this PR should be held up on a final direction there.